source/index.html.md

```diff
@@ -1966,8 +1966,9 @@ Field Name | Type | Description | Units
 }
 ```
 
-Returns details for one or more of your orders. Only opened or partially filled orders will be returned by this endpoint. 
-As soon as an order has been fully matched or canceled, it will not be included in the response. 
+Returns details for one or more of your orders. Purpose of this endpoint is to return opened or partially filled orders. 
+As soon as an order has been fully matched or canceled, it should not be included in the response. 
+Although, for limited time period (up to 60 min) they can still appear there, before they are internally processed. 
```

Comment:
I think this needs proof-reading as I would say
`Although, for limited time period of up to 60 min, they can still appear here before they are internally processed.`

Reply:
Yes...it sounds better :) I will change it. 
And about mentioning the 60 min. If we don't have a good reason to change it, I don't believe we will change it soon. And if, with the new engine, the whole implementation will change - I am happy to update the docs after that....